stdlib/public/Concurrency/Task.cpp

```diff
@@ -1671,9 +1670,10 @@ static void swift_task_asyncMainDrainQueueImpl() {
   }
 #endif
 
-    dispatch_main();
+  dispatch_main();
 #endif
 #endif
+  swift_unreachable("Returned from dispatch_main()");
```

Comment:
This is now in the wrong location.  Please stop pinging me on every single revision.  I will look when I have time.